lib/std/debug.zig

```diff
@@ -1071,13 +1071,8 @@ pub fn readElfDebugInfo(
     parent_mapped_mem: ?[]align(mem.page_size) const u8,
 ) !ModuleDebugInfo {
     nosuspend {
-
-        // TODO https://github.com/ziglang/zig/issues/5525
```

Comment:
This TODO was added in https://github.com/ziglang/zig/commit/fd067fbe8b14c445ed764da15fff5163156126df. The current code no longer uses that workaround, so this TODO no longer seems relevant.